docs/tutorial/README.md

```diff
@@ -6,5 +6,11 @@ tutorials.
 You may already have worked your way through the [getting
 started](?id=getting-started) one.
 
-For a much more in depth encounter with qooxdoo, have a look at the
-comprehensive [tweets app tutorial](twitter/).
+For a much more in depth encounter with qooxdoo, we have some comprehensive tutorials: 
+
+- **qooxdoo tutorial app** ([App](apps://apiviewer) [Code](https://github.com/qooxdoo/qxl.apiviewer))
```

Comment:
Hm, can you check the link? It says apiviewer, not tutorial...

Reply:
sorry was work in progress.